go.sum

```diff
@@ -94,8 +94,6 @@ github.com/PuerkitoBio/purell v1.1.1/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbt
 github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 h1:d+Bc7a5rLufV/sSk/8dngufqelfh6jnri85riMAaF/M=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
-github.com/QvodSoldier/erda-infra v0.0.0-20210412121116-0bde1d7ac1a2 h1:6YRVJlTNFmd5RjELG7CNgUsrYpgxdc1SPGkpl1uLFic=
```

Comment:
@QvodSoldier please check it, i see the erda-infra version is v0.0.0-20210409065408-aba40978f474

Reply:
Already receive ,i 'l check it

Reply:
Already resolve. @sfwn